### PR TITLE
respect the GALAXY_VIRTUAL_ENV 

### PR DESCRIPTION
**Pull request by bgruening**
This will save us 200MB in all containers.

### CODE REVIEW

**Comment by mvdbeek** [comment]
Nice!

File: tasks/galaxy_root.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TODO: following task is kind of a crutch - shouldn't be needed.
 - name: "Run common_startup.sh to create welcome.html from sample"
-  shell: "cd {{ galaxy_server_dir }} && scripts/common_startup.sh"
+  shell: "export GALAXY_VIRTUAL_ENV={{ galaxy_venv_dir }} && cd {{ galaxy_server_dir }} && scripts/common_startup.sh"
```
Comment (mvdbeek) [inline tasks/galaxy_root.yml]: I think you could also write
```
environment: 
  GALAXY_VIRTUAL_ENV: {{ galaxy_venv_dir }}
```

Comment (mvdbeek) [inline tasks/galaxy_root.yml]: you can also drop the cd {{ galaxy_server_dir }} and
```
shell: scripts/common_startup.sh
args: 
  chdir: "{{ galaxy_venv_dir }}"
```

Comment (bgruening) [inline tasks/galaxy_root.yml]: I tried this before, but obviously I had a syntax error. It seems to work now. Thanks this is way better and more ansible like! :)

File: tasks/galaxy_root.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # TODO: following task is kind of a crutch - shouldn't be needed.
 - name: "Run common_startup.sh to create welcome.html from sample"
-  shell: "cd {{ galaxy_server_dir }} && scripts/common_startup.sh"
+  environment: 
+    GALAXY_VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
+  shell: scripts/common_startup.sh
+  arg:
```
Comment (mvdbeek) [inline tasks/galaxy_root.yml]: you're missing a `s` in `args`

**Comment by bgruening** [comment]
ready to go!

**Comment by jmchilton** [comment]
Very nice - thanks a bunch!

**Pull request merged**